src/frontend/komponenter/Behandling/BehandlingContainer.tsx

```diff
@@ -1,18 +1,65 @@
 import * as React from 'react';
 import { FC } from 'react';
-import { useParams } from 'react-router';
+import { Redirect, Route, Switch, useParams } from 'react-router';
 import Inngangsvilkår from './Inngangsvilkår/Inngangsvilkår';
+import Venstremeny from '../Venstremeny/Venstremeny';
+import styled from 'styled-components';
+import { styles } from '../../typer/styles';
+import Høyremeny from '../Høyremeny/Høyremeny';
+import { IBehandlingParams } from '../../typer/routing';
 
-interface Params {
-    behandlingId: string;
-}
+const Container = styled.div`
+    display: flex;
+    height: calc(100vh - 8rem);
+`;
+
+const VenstreMenyWrapper = styled.div`
+    min-width: 10rem;
+    border-right: 2px solid ${styles.farger.navGra40};
+    overflow: hidden;
+`;
+
+const HøyreMenyWrapper = styled.div`
+    border-left: 2px solid ${styles.farger.navGra40};
+    overflow-x: hidden;
+    overflow-y: scroll;
+`;
+
+const InnholdWrapper = styled.div`
+    padding: 1rem;
+    flex: 1;
+    overflow: auto;
+`;
 
 const BehandlingContainer: FC = () => {
-    const { behandlingId } = useParams<Params>();
+    const { behandlingId } = useParams<IBehandlingParams>();
 
     return (
         <>
-            <Inngangsvilkår behandlingId={behandlingId} />
+            <Container>
+                <VenstreMenyWrapper>
```

Comment:
Kanske VenstreMenyWrapper borde flyttas til venstremeny... samma med HøyreMenyWrapper flyttas in i Høyremeny..

Reply:
Det ble gjort kun for å håndtere stylingen av de 3 forskjellige kolonnene på ett sted, her i denna fila, tror jeg